dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -242,28 +224,36 @@ public static void cleanStatePrivate1(DropwizardTestSupport<DockstoreWebserviceC
      *
      * @param support    reference to testing instance of the dockstore web service
      * @param configPath
-     * @throws Exception
      */
-    private static void cleanStatePrivate1(DropwizardTestSupport<DockstoreWebserviceConfiguration> support, String configPath)
-        throws Exception {
-        Application<DockstoreWebserviceConfiguration> application = support.getApplication();
-        application.run("db", "drop-all", "--confirm-delete-everything", configPath);
+    private static void cleanStatePrivate1(DropwizardTestSupport<DockstoreWebserviceConfiguration> support, String configPath) {
+        List<String> migrations = List.of("1.3.0.generated", "1.3.1.consistency", "test.confidential1", "1.4.0", "1.5.0", "test.confidential1_1.5.0", "1.6.0", "1.7.0", "1.8.0", "1.9.0", "1.10.0", "1.11.0", "1.12.0", "1.13.0");
+        dropAllAndRunMigration(migrations, support.getApplication(), configPath);
+    }
+
+    public static void runMigration(List<String> migrations, Application<DockstoreWebserviceConfiguration> application,
+        String configPath) {
+        String s = migrations.stream().collect(Collectors.joining(","));
+        LOG.error("RUN MIGRATION " + s);
```

Comment:
I don't think you should log this and the next line as ERROR -- might get confusing when looking for legit errors.

Reply:
> I don't think you should log this and the next line as ERROR -- might get confusing when looking for legit errors.

Doh, I reintroduced this debugging code when I reverted the other changes.  Removing....